physx-sys/src/physx_api.cpp

```diff
@@ -212,6 +240,14 @@ extern "C"
         return new RaycastFilterTrampoline(callback, userData);
     }
 
+    PxAllocatorCallback *create_alloc_callback(
+        AllocCallback alloc_callback,
+        DeallocCallback dealloc_callback,
+        void *userdata
+    ) {
+        return new CustomAllocatorTrampoline(alloc_callback, dealloc_callback, userdata);
```

Comment:
maybe we should have a way of destroying this? :-)

Reply:
We do. It's a PxAllocatorCallback, just ->Release() it.